cluster/images/obs-csi-plugin/entrypoint.sh

```diff
@@ -39,16 +30,24 @@ else
     exit
 fi
 
-echo "Starting deploy obs csi-plugin...."
+echo "Starting install obsfs...."
 mkdir -p /dev/csi-tool/
 tar -zxvf /root/$fileName.tar.gz -C /dev/csi-tool/
 $HOST_CMD cp -r /dev/csi-tool/$fileName/. ./
 $HOST_CMD bash install_obsfs.sh
 
-echo "Starting run obs socket server...."
+echo "Starting install obs socket-server...."
+rm -rf /dev/csi-tool/socket-server
+rm -rf /dev/csi-tool/connector.sock
```

Comment:
why delete this  file ?

Reply:
This script use for updating socket-server, remove the old files and copy the new files.

Reply:
This may cause "file busy error" if did not remove the old files.